react-web/src/pages/subscription/history/SubscriptionHistory.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+import tz from "dayjs/plugin/timezone";
+
+import TableComponent from "components/Table/Table";
+import { fetchData } from "api/api";
+import { Subscription } from "../Subscription.interface";
+import Toast from "components/Toast/Toast";
+
+dayjs.extend(utc)
+dayjs.extend(tz)
+
+const SubscriptionHistory = () => {
+    const navigate = useNavigate();
+    const [data, setData] = useState<Array<Subscription>>([]);
+    const [skipPageReset] = useState(false);
+    const [errorToast, setErrorToast] = useState<{display: boolean; statusText?: string; message?: string;}>({display: false});
```

Comment:
same, why state ?!

Reply:
Added the handleError back in , and used this state variable. Hence retained as is